### PR TITLE
Feature: Clipboard component

### PR DESCRIPTION
**Pull request by joshsadam**
New Viral Component to render a text field and button, the field is a read-only field that has the value set to the value to copy.  When the user clicks the button, the value is copied to the system clipboard.
 - eye icon to toggle the visibility of the value.
 - fallback for older browsers using `document.execCommand("copy")`

Update `_access_token_section` with new clipboard component

### CODE REVIEW

**Review by ericenns — CHANGES_REQUESTED** [review]
Looks good in both light and dark mode, just a few minor changes requested.

File: app/components/clipboard_component.html.erb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<%= render Viral::BaseComponent.new(tag: 'div', **system_arguments, data: {
+  controller: "clipboard-component",
+  "clipboard-component-item-value": value }) do %>
+  <div class="relative mr-2 grow">
+    <input
+      type="password"
+      data-clipboard-component-target="input"
+      class="
```
Comment (ericenns) [inline app/components/clipboard_component.html.erb:8]: I'd also like this to use mono spaced font like in the previous pr

File: app/components/clipboard_component.html.erb [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<%= render Viral::BaseComponent.new(tag: 'div', **system_arguments, data: {
+  controller: "clipboard-component",
+  "clipboard-component-item-value": value }) do %>
+  <div class="relative mr-2 grow">
+    <input
+      type="password"
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply app/components/clipboard_component.html.erb]: Updated in c4c5ac2

**Review by ericenns — APPROVED** [review]
Looks good to me!

**Pull request merged**